modules/cpmso-yaml-failure-domain-vsphere.adoc

```diff
@@ -43,5 +43,6 @@ spec:
 
 [IMPORTANT]
 ====
-Each `failureDomains.platform.vsphere.name` field value in the `ControlPlaneMachineSet` CR must match the corresponding value defined in the `failureDomains.name` field of the cluster-wide infrastructure CRD. Currently, the `vsphere.name` field is the only supported failure domain field that you can specify in the `ControlPlaneMachineSet` CR.
-====
+Each `failureDomains.platform.vsphere.name` field value in the `ControlPlaneMachineSet` CR must match the corresponding value defined in the `failureDomains.name` field of the cluster-wide infrastructure CRD.
+Currently, the `vsphere.name` field is the only supported failure domain field that you can specify in the `ControlPlaneMachineSet` CR.
+====
```

Comment:
LGTM , if it helps we can add below  to get failure domain name from infrastructure CRD.
`oc get infrastructure cluster -o=jsonpath={.spec.platformSpec.vsphere.failureDomains[0].name}`

Reply:
That's a great thing to add, thanks!